cloud/docker/_docker.py

```diff
@@ -562,6 +562,8 @@ def _human_to_bytes(number):
 
     if isinstance(number, int):
         return number
+    if number.isdigit():
+        return int(number)
```

Comment:
`number.isdigit()` should be false when `number == "15Mb"` or `number == "15K"`
And `number.isdigit()` is added so that 
the default value `0` or other input value in `string` without suffix
to not be processed by the code below (which requires the string to contain one of the suffixes or raise the error specified in #3078)